webapp/views/wpt-results.js

```diff
@@ -365,6 +367,16 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
     ];
   }
 
+  connectedCallback() {
+    super.connectedCallback();
+    this.addEventListener('triagemetadata', this.handleReloadPendingMetadata.bind(this));
+  }
+
+  disconnectedCallback() {
+    this.removeEventListener('triagemetadata', this.handleReloadPendingMetadata.bind(this));
```

Comment:
This `removeEventListener` call won't work, because calling `this.handleReloadPendingMetadata.bind(this)` twice will return different functions, and `removeEventListener` needs exactly the same arguments as `addEventListener` to work.

If removing the event listener is important, you'll need to store the callback (`this.handleReloadPendingMetadata.bind(this)`) somewhere are pass it in here.

Reply:
Good catch!